packages/main/src/components/AnalyticalTable/index.tsx

```diff
@@ -263,13 +308,21 @@ const AnalyticalTable: FC<TableProps> = forwardRef((props: TableProps, ref: Ref<
               <header {...headerProps} role="rowgroup">
                 {headerGroup.headers.map((column, index) => (
                   <ColumnHeader
+                    id={column.id}
                     {...column.getHeaderProps()}
-                    isLastColumn={index === columns.length - 1}
+                    isLastColumn={index === cols.length - 1}
                     groupable={props.groupable}
                     sortable={props.sortable}
                     filterable={props.filterable}
                     onSort={props.onSort}
                     onGroupBy={onGroupByChanged}
+                    onDragStart={(e) => handleDragStart(column, e)}
```

Comment:
please don´t use inline functions as they are causing unnecessary rerenders.
The way to go for this specific use case is to create a `onDragStart` function inside of ColumnHeader which then calls `props.onDragStart(column, e)`